packages/jest-cli/src/watch.js

```diff
@@ -68,31 +71,55 @@ export default function watch(
   });
 
   const updateConfigAndRun = ({
+    bail,
+    collectCoverage,
+    collectCoverageFrom,
+    collectCoverageOnlyFrom,
+    coverageDirectory,
+    coverageReporters,
     mode,
+    notify,
+    notifyMode,
+    onlyFailures,
+    reporters,
     testNamePattern,
     testPathPattern,
     updateSnapshot,
+    verbose,
   }: {
+    bail?: boolean,
+    collectCoverage?: boolean,
+    collectCoverageFrom?: Array<string>,
+    collectCoverageOnlyFrom?: {[key: string]: boolean},
+    coverageDirectory?: string,
+    coverageReporters?: Array<string>,
     mode?: 'watch' | 'watchAll',
+    notify?: boolean,
+    notifyMode?: string,
+    onlyFailures?: boolean,
+    reporters?: Array<ReporterConfig>,
     testNamePattern?: string,
     testPathPattern?: string,
     updateSnapshot?: SnapshotUpdateState,
+    verbose?: boolean,
```

Comment:
Looks like this could be replaced with `Options` type from `update_global_config.js`?

Reply:
@thymikee By now this is correct — the patch evolved organically, but now we're done! Centralizing.